src/Symfony/Component/DependencyInjection/ContainerInterface.php

```diff
@@ -21,7 +22,7 @@
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
  */
-interface ContainerInterface
+interface ContainerInterface extends PsrContainerInterface
```

Comment:
@jvasseur yes it does
the issue is that `$foo instanceof A` implies that calling `$foo->test($anotherInstanceOfA)` is allowed.
Since `$anInstanceofB instanceof A`, calling `$anInstanceofB->test($anotherInstanceOfAButNotB)` must thus be allowed.